README.md

```diff
@@ -0,0 +1,3 @@
+[Short group signatures](https://crypto.stanford.edu/~xb/crypto04a/groupsigs.pdf) by Boneh, Boyen, and Shachum
+and later improved in [ASM](http://web.cs.iastate.edu/~wzhang/teach-552/ReadingList/552-14.pdf) as BBS+ and touched on again
+in section 4.3 in [CDL](https://eprint.iacr.org/2016/663.pdf).
```

Comment:
Please add at least enough information to be able to build and run the tests.

Reply:
Ah yes - as you've seen I added a makefile and the CI runs the unit-test target. Should I add something like

```
## Testing

To run the unit tests, execute the `unit-tests` Makefile target.
```

?

Reply:
Sounds good